lib/permutations.rb

```diff
@@ -1,4 +1,20 @@
 
 def permutations?(string1, string2)
```

Comment:
Really good attempt at this one as well.  You're on the path to a solution, just not *quite* there yet.  Take a look at my comments and let me know where I can clarify better.